homeassistant/components/homeassistant_hardware/silabs_multiprotocol_addon.py

```diff
@@ -245,6 +259,7 @@ def __init__(self, config_entry: config_entries.ConfigEntry) -> None:
         # If we install the add-on we should uninstall it on entry remove.
```

Comment:
This comment looks a bit mal-placed. Not caused by this PR obviously, but still.